phase1-coordinator/src/coordinator.rs

```diff
@@ -328,6 +328,8 @@ pub struct Coordinator {
     state: Arc<RwLock<CoordinatorState>>,
     /// The source of time, allows mocking system time for testing.
     time: Arc<dyn TimeSource>,
+    /// Callback to call after aggregation is done
+    aggregation_callback: Arc<dyn Fn(Vec<Participant>) -> () + Send + Sync>,
```

Comment:
what is the vec of participants in the callback function argument for?

Reply:
oh I see, might be good to cross reference this with `set_aggregation_callback` method.

Reply:
Maybe it would be good to place this vec within a struct, `AggregationCompleteInfo` or something similar, to document what the fields are and also support easy extension if required?